FWCore/PythonUtilities/python/LumiList.py

```diff
@@ -73,12 +77,12 @@ def __init__(self, filename = None, lumis = None, runsAndLumis = None, runs = No
         if isinstance(runsAndLumis, list):
             queued = {}
             for runLumiList in runsAndLumis:
-                for run, lumis in runLumiList.items():
+                for run, lumis in list(runLumiList.items()):
```

Comment:
Here and in many cases below: why is a `list` created here? The iteration should work fine in both py2 and py3 with the return value of `items()` (or `keys()`). (or am I missing something?)